operations/helm/charts/mimir-distributed/templates/NOTES.txt

```diff
@@ -32,4 +32,4 @@ Ingress is not enabled, see the {{ if $.Values.enterprise.enabled }}gateway{{ el
 From inside the cluster:
   http://{{ include "mimir.fullname" $ }}-{{ if .Values.enterprise.enabled }}gateway{{ else }}nginx{{ end }}.{{ .Release.Namespace }}.svc:{{ $gateway.service.port }}{{ template "mimir.prometheusHttpPrefix" $ }}
 
-**IMPORTANT**: Always consult the CHANGELOG.md at https://github.com/grafana/mimir/blob/main/operations/helm/charts/mimir-distributed/CHANGELOG.md file to learn about breaking changes that require action during upgrade.
```

Comment:
I just ran `helm upgrade` and a similar message showed up. It made me think whether or not this should be there. I already did the upgrade, isn't it too late to look at the changelog? 😄 

@krajorama

Reply:
It's a good reminder anyway I think